sdk/communication/communication-short-codes/src/shortCodesClient.ts

```diff
@@ -81,7 +75,7 @@ export class ShortCodesClient {
       options.userAgentOptions.userAgentPrefix = libInfo;
```

Comment:
The constructor in `src/generated/src/shortCodesClientContext.ts` already handles user agent option so don't need it here any more.